concourse/pipeline.yml

```diff
@@ -1,453 +1,287 @@
 ---
 resources:
-  - name: repo
-    type: git
-    source:
-      uri: {{app-url}}
-      branch: {{app-branch}}
-      private_key: {{github-private-key}}
-  - name: version
-    type: git
-    source:
-      uri: {{app-url}}
-      branch: version
-      private_key: {{github-private-key}}
-  - name: tools
-    type: git
-    source:
-      uri: {{tools-scripts-url}}
-      branch: {{tools-branch}}
-  - name: m2
-    type: docker-image
-    source:
-      repository: springcloud/spring-pipeline-m2
+- name: repo
+  type: git
+  source:
+    uri: ((app-url))
```

Comment:
`{{ }}` is [now deprecated](https://concourse.ci/downloads.html#v320) since curly braces are actually YAML syntax keywords (YAML flow syntax specifically `source: {uri: https://...}`).  They never should have used it and are now in alignment with most other YAML templating systems.